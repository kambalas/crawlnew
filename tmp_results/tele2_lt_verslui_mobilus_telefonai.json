[
  {
    "name": "Xiaomi Redmi 14C 4G (128 GB)",
    "price": "139.67",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 Pro+ 5G (512 GB)",
    "price": "313.22",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 4G (256 GB)",
    "price": "164.46",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 4G (128 GB)",
    "price": "147.93",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 Pro 4G (256 GB)",
    "price": "205.79",
    "error": false
  },
  {
    "name": "Xiaomi 14T Pro 5G (512 GB)",
    "price": "495.04",
    "error": false
  },
  {
    "name": "Xiaomi 15 Ultra 5G (512 GB)",
    "price": "1238.84",
    "error": false
  },
  {
    "name": "Xiaomi 15 5G (512 GB)",
    "price": "825.62",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 5G (256 GB)",
    "price": "247.11",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 5G (128 GB)",
    "price": "222.31",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 14 Pro 5G (256 GB)",
    "price": "329.75",
    "error": false
  },
  {
    "name": "Xiaomi 14T 5G (256 GB)",
    "price": "453.72",
    "error": false
  },
  {
    "name": "Xiaomi Redmi 13 (256 GB)",
    "price": "164.46",
    "error": false
  },
  {
    "name": "Xiaomi Redmi 13 (128 GB)",
    "price": "114.88",
    "error": false
  },
  {
    "name": "Xiaomi Redmi 13C 5G (128 GB)",
    "price": "164.46",
    "error": false
  },
  {
    "name": "Xiaomi Redmi A3 (64 GB)",
    "price": "99.17",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 13 Pro+ 5G (512 GB)",
    "price": "362.81",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 13 Pro 5G (256 GB)",
    "price": "222.31",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 13 Pro 4G (256 GB)",
    "price": "205.79",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 13 5G (256 GB)",
    "price": "189.26",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 13 5G (128 GB)",
    "price": "164.46",
    "error": false
  },
  {
    "name": "Xiaomi Redmi Note 13 (256 GB)",
    "price": "147.93",
    "error": false
  }
]