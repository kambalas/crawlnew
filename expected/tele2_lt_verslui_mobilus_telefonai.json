[
  {
    "name": "Honor Magic7 Lite 5G (512 GB)",
    "price": "32975",
    "error": false
  },
  {
    "name": "Honor Magic7 PRO 5G (512 GB)",
    "price": "1073594",
    "error": false
  },
  {
    "name": "Honor 200 smart 5G (256 GB)",
    "price": "13123",
    "error": false
  },
  {
    "name": "Honor Magic V3 5G (512 GB)",
    "price": "16521486",
    "error": false
  },
  {
    "name": "Honor Magic 6 Lite5G",
    "price": "18926164",
    "error": false
  }
]